end-to-end-test/local/specs/annotation-filter-menu.spec.js

```diff
@@ -245,7 +257,7 @@ describe('alteration filter menu', function() {
                 clickCheckBoxStudyView('Putative passengers');
             });
 
-            it('filters tables when unchecking when unchecking unknown oncogenicity checkbox', () => {
+            it.skip('filters tables when unchecking when unchecking unknown oncogenicity checkbox', () => {
```

Comment:
@pvannierop it's the tests that are skipped here which relate to annotation filtering.  it kind of makes sense that these would be broken given that whatever script was running to annotate fusions now has to run on structural variant data?  if you need to build

Reply:
@alisman That the numbers are not correct for the Struct Vars makes sense indeed. I do not know whether the Struct Var implementation also allows for _custom driver annotations_? This feature is what drives the oncogenicity filtering in said menu. Also I do not know how the Struct Vars in _study_es_0_ are supposed to filter according to somatic/germline status.